www/pfSense-pkg-squid/files/usr/local/pkg/squid.inc

```diff
@@ -2072,19 +2072,15 @@ function squid_generate_rules($type) {
 	$ssl_port = ($squid_conf['ssl_proxy_port'] ? $squid_conf['ssl_proxy_port'] : 3129);
 
 	$fw_aliases = filter_generate_aliases();
-	if (strstr($fw_aliases, "pptp =")) {
-		$PPTP_ALIAS = "\$pptp";
-	} else {
-		$PPTP_ALIAS = "\$PPTP";
-	}
 	if (strstr($fw_aliases, "PPPoE =")) {
 		$PPPOE_ALIAS = "\$PPPoE";
 	} else {
 		$PPPOE_ALIAS = "\$pppoe";
 	}
 
 	// define ports based on transparent options and ssl filtering
-	$pf_rule_port = ($squid_conf['ssl_proxy'] == "on" ? "{80,443}" : "80");
+	$pf_nat_ports = ($squid_conf['ssl_proxy'] == "on" ? "{80,443}" : "80");
+	$pf_rule_ports = "{{$port},{$ssl_port}}";
```

Comment:
- ```$pf_nat_ports``` is for transparent proxy
- ```$pf_rule_ports``` is for the defined proxy interfaces (nontransparent) 

(with, or without HTTPS/SSL Interception)